crates/nargo/src/cli/prove_cmd.rs

```diff
@@ -27,6 +28,14 @@ pub(crate) fn run(args: ArgMatches) -> Result<(), CliError> {
 /// So when we add witness values, their index start from 1.
 const WITNESS_OFFSET: u32 = 1;
 
+// A map from the fields in an TOML/JSON file
+// which correspond to some ABI to their values
+pub type AbiMap = BTreeMap<String, InputValue>;
```

Comment:
```suggestion
pub type InputMap = BTreeMap<String, InputValue>;
```

I'm concerned about possible confusion between this and the mapping from abi params and the witness indices which they are mapped to. This name would be clearer imo.